gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/proc/KillDagProcTest.java

```diff
@@ -215,8 +216,6 @@ public void killDagNode() throws IOException, URISyntaxException, InterruptedExc
             .getInvocations()
             .stream()
             .filter(a -> a.getMethod().getName().equals("cancelJob"))
-            .filter(a -> ((Properties) a.getArgument(1))
```

Comment:
the first job was launched, but we are cancelling the 3rd job, so the cancelParams will not have  "SPEC_PRODUCER_SERIALIZED_FUTURE". this is realized when i fixed the test mock behavior above